api/osrd_infra/models/rolling_stock.py

```diff
@@ -64,6 +64,16 @@ class RollingStock(models.Model):
     loading_gauge = models.CharField(max_length=16, choices=[(x.value, x.name) for x in LoadingGaugeType])
     image = models.ImageField(null=True, blank=True)
 
+    # Catenary related
+    electric_only = models.BooleanField(
+        help_text=_("If true, the train can only use tracks with compatible catenaries")
+    )
+    compatible_voltages = ArrayField(
+        models.PositiveIntegerField(),
+        blank=True,
+        help_text=_("A list of compatible voltage (in V)"),
+    )
```

Comment:
Do we have cases where a rolling stock has compatible voltages but isn't electric only

Reply:
Yes, for some rolling stocks they can use electricity but it's not their only mode. They can go everywhere but eventually we'll need the compatible voltages to pick the correct effort curve.